proto/responses.proto

```diff
@@ -195,9 +195,23 @@ message AccountProofsResponse {
 message AccountStateHeader {
     // Account header.
     account.AccountHeader header = 1;
+
     // Values of all account storage slots (max 255).
     bytes storage_header = 2;
-    // Account code, returned only when none of the request's code commitments match with the
-    // current one.
+
+    // Account code, returned only when none of the request's code commitments match
+    // the current one.
     optional bytes account_code = 3;
+
+    // Storage slots information for this account
+    repeated StorageSlotMapProof storage_slots = 4;
```

Comment:
Should the name be `storage_maps` since this is only for storage map slots?

Also, this is probably fine for now, but ideally we'd return a single batch proof per map. In cases when more than one key is requested per map, this could save quite a bit on bandwidth. Let's create an issue for this.

Reply:
Changed!